src/pyroboplan/planning/prm.py

```diff
@@ -123,20 +133,35 @@ def construct_roadmap(self, sample_generator=None):
             if check_collisions_at_state(self.model, self.collision_model, q_sample):
                 continue
 
+            radius = self.options.max_neighbor_radius
+            k = self.options.max_neighbor_connections
```

Comment:
can this go in an `else` below?

Reply:
I updated the comments and made it more obvious that PRM* scales these original values, hopefully that's more clear? Otherwise we just end up with the same assignments in both blocks.